src/components/accordion/index.md

```diff
@@ -65,6 +65,8 @@ There are 2 ways to use the accordion component. You can use HTML or, if you’r
 
 The accordion component uses JavaScript. When JavaScript is not available, users will see all the content displayed with the section labels as headings.
 
+The 'id' attribute on the the parent div of the accordion is a required attribute. It must be unique across the domain of your service to persist the expanded state of the accordion (the state individual instances of the component persists across page loads using [session storage](https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage)).
```

Comment:
Is it possible to make this into the active voice without changing its meaning? For example - You'll require the 'id' attribute from the parent div of the accordion. 
Also, I'm unclear what you mean in the last sentence in brackets.

```suggestion
The 'id' attribute on the the parent div of the accordion is a required attribute. It must be unique across the service's domain to maintain the persistent expanded state of the accordion (the state individual instances of the component persists across page loads using [session storage](https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage)).
```